CHANGELOG.md

```diff
@@ -0,0 +1,192 @@
+# CHANGELOG
+
+### 3.0-alpha 
```

Comment:
nitpick: it might be more consistent to change the heading level for all versions from 3 to 2